src/main/java/mate/academy/bookstore/repository/RoleRepository.java

```diff
@@ -0,0 +1,10 @@
+package mate.academy.bookstore.repository;
+
+import java.util.Set;
+import mate.academy.bookstore.model.Role;
+import mate.academy.bookstore.model.Role.RoleName;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface RoleRepository extends JpaRepository<Role, Long> {
+    Set<Role> getAllByName(RoleName name);
+}
```

Comment:
Roles in your database are unique, so the result of calling this method will be a single Role, so, I think, it's better to use Optional<Role> here (we are not sure that we will GET the role, so we will try to FIND it)

Reply:
agree,let's return `Optional` here